docs/INSTALLATION.md

```diff
@@ -35,7 +35,11 @@ Update the `values.yaml` based on what you provisioned as part of the prerequisi
      * For more details, please refer to the **Volumes** section of the [configuration guide](CONFIGURATION.md).
      * You can also refer to our examples of AWS storage: [Local storage - utilizing AWS EBS-backed volumes](docs/examples/storage/aws/LOCAL_STORAGE.md) and [Shared storage - utilizing AWS EFS-backed filesystem](docs/examples/storage/aws/SHARED_STORAGE.md)    
     * Bitbucket needs a dedicated NFS server providing persistence for a shared home. Prior to installing the Helm chart, a suitable NFS shared storage solution must be provisioned. The exact details of this resource will be highly site-specific, but you can use this example as a guide: [Implementation of an NFS Server for Bitbucket](docs/examples/storage/nfs/NFS.md)
-
+ 
+ * **Bitbucket Secrets**: If you specify the names of the [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) that contain the Bitbucket license key and the Bitbucket sysadmin credentials, you will be able to bypass the license and the admin user configurations during the Bitbucket setup. To do this, update the `bitbucket` stanza in the `values.yaml` file:
+    * Update the `secretName` that is under `license` and the `secretName` that is under `sysadminCredentials`. You can provide the Bitbucket license key and sysadmin credentials via a Kubernetes secret, with the secret name specified here.
+    
+   * [Create the secrets](https://kubernetes.io/docs/concepts/configuration/secret/#creating-a-secret) with the keys defined in the `values.yaml` file.
```

Comment:
It is a bit chicken and egg problem here, but I would probably first put the `Create the secrets` step and then update `secretName` bullet point.